src/components/modals/WalletModal/index.tsx

```diff
@@ -195,18 +194,7 @@ const WalletModal: React.FC = () => {
 
       if (option.connector === injected) {
         if (!(window.web3 || window.ethereum)) {
-          if (option.name === 'MetaMask') {
-            return (
-              <Option
-                icon={MetamaskIcon}
-                key={key}
-                onClick={() => window.open('https://metamask.io/')}
```

Comment:
I think providing the link is to metamask is quite useful. I would not want to remove it.

I would prefer to disable the button, until the user agrees to terms and then still continue to show this link.

Reply:
That could end up being kind of confusing: The user has to accept the T&C to enable a button which is a link to install Metamask (accepting the T&C to install an external plugin is weird IMO), which - after installing it- will be the button to connect to Metamask... that's convoluted.

A better (or just simpler) option could be a line of text at the bottom, something like "Don't have a wallet? Try Metamask!" could do it.

Reply:
>A better (or just simpler) option could be a line of text at the bottom, something like "Don't have a wallet? Try Metamask!" could do it.

you are right. Probably that is simpler.

Reply:
Should look like this now.

![2021-04-27 15_15_14-Gnosis Auction](https://user-images.githubusercontent.com/4015436/116292349-f9d8b400-a76b-11eb-9e96-6c79f529a9ee.png)